vnext/ReactWindowsCore/Utils.cpp

```diff
@@ -4,17 +4,17 @@
 #include "Utils.h"
 #include <regex>
 
-using namespace std;
+using std::string;
 
 namespace Microsoft::React {
 
 Url::Url(const string &source) {
-  //      ( 1 )              ( 2 )   ( 3 (4) )   ( 5 )    ( 6 (7) )
-  regex expression("(http|https|ws|wss)://([^:/\\?]+)(:(\\d+))?(/[^\\?]*)?(\\?(.*))?$");
-  //     protocol             host       port     path        query
-  cmatch match;
+  //                           ( 1 )              ( 2 )   ( 3 (4) )   ( 5 )    ( 6 (7) )
+  std::regex expression("(http|https|ws|wss)://([^:/\\?]+)(:(\\d+))?(/[^\\?]*)?(\\?(.*))?$");
+  //                          scheme              host       port      path      query
+  std::cmatch match;
   int index = 0;
-  if (regex_match(source.c_str(), match, expression)) {
+  if (std::regex_match(source.c_str(), match, expression)) {
     ++index;
     this->scheme = string(match[index].first, match[index].second);
```

Comment:
instea of keeping index and advancing it, why not just reference match[3].first, etc?
better yet:
constexpr int scheme = 1;
constexpr int host = 2;
constexpr int port = 3;
...
(or as enum class and cast to int)
then reference match[port], that way it's really clear what index you're accessing on every line